plugin-server/src/utils/db/db.ts

```diff
@@ -418,9 +418,10 @@ export class DB {
         }
     }
 
-    public async fetchPerson(teamId: number, distinctId: string): Promise<Person | undefined> {
-        const selectResult = await this.postgresQuery(
-            `SELECT
+    public async fetchPerson(teamId: number, distinctId: string): Promise<Person | undefined>
+    public async fetchPerson(teamId: number, distinctId: string, client: PoolClient): Promise<Person | undefined>
```

Comment:
the overload is unnecessary given the return type is the same and we're just handling one extra arg being there or not.